File: starters/Rmsbolt.java
// Java rmsbolt starter file

// Local Variables:
// rmsbolt-command: "javac"
// rmsbolt-filter-directives: t
// End:

public class Rmsbolt {
	public static int isRMS(char in) {
		switch (in) {
		case 'R':
			return 1;
		case 'M':
			return 2;
		case 'S':
			return 3;
		default:
			return 0;
		}
	} 

	public static void main(String[] args) {
		char a = 1 + 1;
		if (isRMS(a) == 0)
			System.out.println(a);
	}
}

File: starters/rmsbolt.nim
import strutils

# Nim rmsbolt starter file

# Local Variables:
# rmsbolt-command: "nim c --opt:size"
# End:



proc isRMS(a: char): uint8 {.exportc.} =
  case a
  of 'R': result = 1
  of 'M': result = 2
  of 'S': result = 3
  else: result = 0

proc main(): void =
  const a: char = cast[char](1 + 1)
  if (isRMS(a) != 0): 
    echo "$#" % $a
